docs/src/main/sphinx/release/release-450.md

```diff
@@ -0,0 +1,68 @@
+# Release 450 (7 Jun 2024)
+
+## General
+
+* Add support for defining a number of rows to be pushed from a filter into
```

Comment:
This should describe the user-visible impact. "pushed from a filter into values" doesn't mean anything for an end user.

Reply:
I'm honestly not too sure what the feature is doing, and it wasn't documented, so there's not much to fall back on 😔

Reply:
Should we reword around performance and make it higher level?

Reply:
I think we should remove this item. It's too low level and doesn't have much direct visible impact to end users.